### PR TITLE
Removed charwidth setting and added output.info method

### PR DESCRIPTION
**Pull request by jlstevens**
This screenshot illustrates the `output.show` method which is the replacement for using the `%output` magic without any arguments:

image

The charwidth setting is now removed as it was just confusing and unused cruft.

### CODE REVIEW

**Comment by jlstevens** [comment]
Maybe `output.show()` is confusing and it could be called `output.settings()`? Though maybe that might be expected to return the settings as a dict instead of just showing them?

**Comment by jlstevens** [comment]
We decided both `opts` and `output` should call this `info`.

File: holoviews/core/options.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def __call__(self, *args, **kwargs):
     def clear(self, clone=False):
         return self.obj.opts(clone=clone)
 
-    def show(self):
+    def info(self):
```
Comment (philippjfr) [inline holoviews/core/options.py]: I'd like to expose whether to show defaults here and add a small docstring. I'd be happy to do so in another PR though.

Comment (jlstevens) [reply holoviews/core/options.py]: Agreed. Good idea for a separate PR.
